homeassistant/components/light/knx.py

```diff
@@ -157,17 +153,18 @@ def supported_features(self):
         if self.device.supports_brightness:
             flags |= SUPPORT_BRIGHTNESS
         if self.device.supports_color:
-            flags |= SUPPORT_RGB_COLOR
+            flags |= SUPPORT_COLOR
         return flags
 
     async def async_turn_on(self, **kwargs):
         """Turn the light on."""
         if ATTR_BRIGHTNESS in kwargs:
             if self.device.supports_brightness:
                 await self.device.set_brightness(int(kwargs[ATTR_BRIGHTNESS]))
-        elif ATTR_RGB_COLOR in kwargs:
+        elif ATTR_HS_COLOR in kwargs:
             if self.device.supports_color:
-                await self.device.set_color(kwargs[ATTR_RGB_COLOR])
+                await self.device.set_color(color_util.color_RGB_to_hs(
+                    *kwargs[ATTR_HS_COLOR]))
```

Comment:
This converts in the wrong direction.

Maybe @Julius2342 will know if this call ignores the default full brightness that `color_util.color_hs_to_RGB()` sets.